web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -135,6 +139,20 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
   const userProfile = selectUserProfile(state);
   const isAdmin = userProfile?.role?.includes('ADMIN');
 
+  useQueryParameters([
+    {
+      name: 'add',
```

Comment:
The only state I see that we can track with query parameters on this page is whether the "ADD REVIEW PERIOD" button was clicked.